semeio/jobs/rft/utility.py

```diff
@@ -0,0 +1,265 @@
+import os
+import datetime
+import argparse
+
+from ecl.rft import EclRFTFile
+from ecl.grid import EclGrid
+from semeio.jobs.rft.zonemap import ZoneMap
+
+
+def strip_comments(line):
+    return line.partition("--")[0].rstrip()
+
+
+def existing_directory(path):
+    if not os.path.exists(path):
```

Comment:
maybe use `os.path.is_dir()` instead of `os.path.exists`?

Reply:
thanks, fixed!